doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -52,6 +52,18 @@ This is a much faster way than waiting for the subscriber node to notice that th
 
 To enable persistent subscriptions, set the :kconfig:option:`CONFIG_CHIP_PERSISTENT_SUBSCRIPTIONS` Kconfig option.
 
+The time that it takes to re-establish all subscriptions and CASE sessions depends on the number of sessions that can be recovered simultaneously.
+If the number of sessions and subscriptions to recover is greater than the maximum, the device will re-establish as many as possible and schedule re-establishment of the rest after a specified time interval.
+Since the wait interval between the following re-establishment attempts impacts the overall time to recover all device connections, so decreasing it can be beneficial for the user experience.
```

Comment:
```suggestion
Since the wait interval between the following re-establishment attempts impacts the overall time to recover all device connections, decreasing it can be beneficial for the user experience.
```
or

```suggestion
The wait interval between the following re-establishment attempts impacts the overall time to recover all device connections, so decreasing it can be beneficial for the user experience.
```

Reply:
Good catch. Either option is good.